Source/PeleLMeX_EBUserDefined.H

```diff
@@ -25,39 +25,5 @@ EBUserDefined(
   Abort("In default EBUserDefined function! Shouldn't be here. Copy and edit "
```

Comment:
Why not also include `EBUserDefined` in ProbIBC? That way everything case-specific would be in the same place, making things simpler if we move toward compiling multiple cases at the same time.